src/Android/Accessibility/AccessibilityActivity.cs

```diff
@@ -78,6 +80,21 @@ protected override void OnActivityResult(int requestCode, [GeneratedEnum] Result
             Finish();
         }
 
+        private void HandleIntent(Intent callingIntent, int requestCode)
+        {
+            if(callingIntent?.GetBooleanExtra("autofillTileClicked", false) == true)
```

Comment:
Do we need the `=== true` here? It also seems like you are doing a null check with `?.` here, so I would write it as:

```
callingIntent?.GetBooleanExtra("autofillTileClicked", false) ?? false
```